crates/sui/src/client_commands.rs

```diff
@@ -707,10 +705,9 @@ impl SuiClientCommands {
                     terminate_early,
                     max_tasks: 16,
                 };
-                // todo: automatically get non pruning fullnode url
+                let rpc = rpc_url.unwrap_or(context.config.get_active_env()?.rpc.clone());
```

Comment:
Let's just use the active env -- i.e. no dedicated `rpc_url` flag that's specific to the `replay` commands -- that's going to introduce unnecessary confusion (Why do these commands get to override the RPC URL, but not other client commands?)